packages/source-strapi/helpers/query.js

```diff
@@ -23,6 +66,15 @@ module.exports = async ({ apiURL, resourceName, jwtToken, queryLimit, isSingleTy
   // Make API request.
   return axios(apiEndpoint, fetchRequestConfig)
     .then(res => res.data)
+    .then(async data => {
+      if (mediaDownloadFolder) {
+        await execSync(`mkdir -p ${mediaDownloadFolder}`)
```

Comment:
Would it be better to use the [`ensureDir()`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md) method from the `fs-extra` module here instead?

Reply:
Yeah great!